rc/filetype/rust.kak

```diff
@@ -6,15 +6,14 @@
 
 hook global BufCreate .*[.](rust|rs) %{
     set-option buffer filetype rust
+    require-module rust
```

Comment:
Why move this line to here? Other language support scripts have it in the `filetype` hook.

Reply:
Why not? We set current filetype, which means we will load the module. There are several hooks for this filetype and I think it's safer this way.

Reply:
Moving this line does not fix a bug, and creates a discrepancy with how other filetype scripts work. If you think it's safer, better modify all the scripts at once in a bug-fix commit instead of having unrelated commits doing it randomly.